docs/tutorials/qsimcirq_gcp.md

```diff
@@ -248,103 +282,88 @@ print("Circuit:")
-
+To avoid incurring charges to your Google Cloud Platform account for the
+resources used in this tutorial:
 
 ### Delete the VM
 
-It is usually recommended to delete the entire project you created for this tutorial, but if you want to continue using the project, you can easily either STOP the VM or Delete it. Select the VM, the either select the STOP square, or the DELETE trash can.
-
-
-
+It is usually recommended to delete the entire project you created for this
+tutorial, but if you want to continue using the project, you can easily either
+STOP the VM or Delete it. Select the VM, the either select the STOP square, or
+the DELETE trash can.
 
 ![alt_text](../images/qsimcirq_gcp/image9.png )
 
-
 ### Delete the project
 
-The easiest way to eliminate billing is to delete the project you created for the tutorial.
-
-**Caution**: Deleting a project has the following effects:
-
-*   **Everything in the project is deleted.** If you used an existing project for this tutorial, when you delete it, you also delete any other work you've done in the project.
-*   **Custom project IDs are lost.** When you created this project, you might have created a custom project ID that you want to use in the future. To preserve the URLs that use the project ID, such as an **```appspot.com```** URL, delete selected resources inside the project instead of deleting the whole project.
-
-If you plan to explore multiple tutorials and quickstarts, reusing projects can help you avoid exceeding project quota limits.
-
-1. In the Cloud Console, [go to the **Manage Resources** page](https://console.cloud.google.com/iam-admin/projects)
-2. In the project list, select the project that you want to delete and then click **Delete** 
-<img src="../images/qsimcirq_gcp/image3.png" width="18"/>
-
-
-3. In the dialog, type the project ID and then click **Shut down** to delete the project.
-
-
-
-## What's next
-
-Additional tutorial examples are available here:
+The easiest way to eliminate billing is to delete the project you created for
+the tutorial.
 
+Be aware that deleting a project has the following effects:
 
+- *Everything in the project is deleted.* If you used an existing project for
+  this tutorial, when you delete it, you also delete any other work you've done
+  in the project.
+- *Custom project IDs are lost.* When you created this project, you might have
+  created a custom project ID that you want to use in the future. To preserve
+  the URLs that use the project ID, such as an `appspot.com` URL, delete
+  selected resources inside the project instead of deleting the whole project.
 
-*   [Cirq examples](https://cirq.readthedocs.io/en/stable/docs/tutorials/examples.html)
+If you plan to explore multiple tutorials and quickstarts, reusing projects can
+help you avoid exceeding project quota limits.
 
-Try out other Google Cloud Platform features for yourself. Have a look at our [tutorials](https://cloud.google.com/docs/tutorials).
```

Comment:
Why are we removing this section? At a minimum, @jrossthomson has indicated that the final line ("Try out other Google Cloud Platform features") is required on all Cloud Solutions documentation.

Reply:
The Cirq examples link is wrong and I think it's confusing to have the user jump to a different section of the site.

> indicated that the final line ("Try out other Google Cloud Platform features") is required on all Cloud Solutions documentation.

This isn't Cloud Solutions documentation 😀 If cross-published they can apply their Cloud editorial requirements. The doc already includes many links to Cloud and I don't think it's an appropriate CTA to send the user to another site :)

Reply:
Sounds good to me.